### PR TITLE
Add scripts to test containerd

### PR DESCRIPTION
**Pull request by EGuesnet**
This PR adds 2 scripts required to test containerd in CI:
- machine.sh instantiates a PowerVS machine with the right configuration and run tests.
- test.sh updates the containerd git repo and performs tests themselves.

### CODE REVIEW

File: test-containerd/machine.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```
Comment (alunsin) [inline test-containerd/machine.sh:1]: General: about the script naming, I would rename the script to be consistent with what it does:
- create a PowerVS instance (aka VM)
- launch the containerd tests on it
- delete the VM when done
How about test_on_powervs ? 
The same remark apply to the usage.

File: test-containerd/machine.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+		*) echo "Unknown argument $1"; usage; exit 1;;
+	esac
+done
+
+# Ensure key and name are fulfilled
+if [ -z $NAME ]; then echo "Name not fulfilled."; usage; exit 1; fi
+if [ -z $SSH_KEY ]; then echo "Key not fulfilled."; usage; exit 1; fi
+
+# Create a machine
+ibmcloud pi instance-create $NAME --image ubuntu_2004_containerd --key-name $SSH_KEY --memory 2 --processor-type shared --processors '0.25' --network 'public-192_168_141_8-29-VLAN_2093' --storage-type tier3
```
Comment (alunsin) [inline test-containerd/machine.sh:45]: `public-192_168_141_8-29-VLAN_2093` : could we move that to a variable to make it easier to change/maintain in the future?

File: test-containerd/test.sh [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+CRUN_VERSION=1.3
+
+RUNC_FLAVOR=$1
+TEST_RUNTIME=$2
+
+cd /home/containerd_test
+cd containerd/
+git pull
+
+# Change size of disk in a test
```
Comment (alunsin) [inline test-containerd/test.sh:14]: Could we tell that this is because of the mkfs error if the size is too small and provide link to the issue?

**Comment by EGuesnet** [comment]
Reviews have been taken into account (name change, more comments, issue linked, NETWORK variable used).

**Review by alunsin — APPROVED** [review]

**Pull request merged**